app/.eslintrc

```diff
@@ -0,0 +1,58 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
```

Comment:
I added `"no-console": 0,` to the .eslintrc file.
https://github.com/mozilla/fxa-content-server/pull/2578/files#r32183274